live-examples/js-examples/number-positive-infinity.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">function checkNumber(bigNumber) {
+	if (bigNumber === Number.POSITIVE_INFINITY) { return "Process number as Infinity" };
```

Comment:
I know we have not decided on a style but, here I reckon the line length itself presents a problem and writing this as, would be preferable:

```
if (bigNumber === Number.POSITIVE_INFINITY) {
    return "Process number as Infinity"
};
```

Reply:
Yup.